kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/authorization/PaymentRequestDocumentPresentationController.java

```diff
@@ -1,29 +1,46 @@
-/*
- * The Kuali Financial System, a comprehensive financial management system for higher education.
- * 
- * Copyright 2005-2014 The Kuali Foundation
- * 
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- * 
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- * 
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
 package edu.arizona.kfs.module.purap.document.authorization;
 
 import java.util.Set;
+
 import edu.arizona.kfs.module.purap.PurapAuthorizationConstants;
+import edu.arizona.kfs.module.purap.document.PaymentRequestDocument;
+
+import org.kuali.kfs.module.purap.PurapConstants.PaymentRequestStatuses;
+import org.kuali.kfs.module.purap.document.service.PurapService;
+import org.kuali.rice.krad.document.Document;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.kew.api.WorkflowDocument;
 import org.kuali.rice.krad.document.Document;
 
 
 public class PaymentRequestDocumentPresentationController extends org.kuali.kfs.module.purap.document.authorization.PaymentRequestDocumentPresentationController {
+	
+    @Override
+    public boolean canEdit(Document document) {
```

Comment:
I don't understand the purpose of this method override. It seems unrelated to this mod. We are also losing some functionality that is in the KFS 6.0 method (see lines 112-114). I would suggest removing this override and retesting. I don't think it is needed.
See https://github.com/ua-eas/eas-ksd-kfs-3.0/commit/a2b290c52611cee064178d031d1bb9792e235a5b for how lock down of bank code and payment method code was achieved in original mod.